src/citationnet/static/pieChart_vega-lite.js

```diff
@@ -0,0 +1,88 @@
+const forChartDiv = document.getElementById("forChartDiv");
+const forChartDivvis = document.getElementById("forChartDivvis");
+const forChartDivloader = document.getElementById("forChartDivloader");
```

Comment:
Would be cleaner to group into an object, e.g.
```suggestion
const chart = {
  container: document.getElementById("forChartDiv"),
  vis: document.getElementById("forChartDivvis"),
  loader: document.getElementById("forChartDivloader")
};
```

Then you can reference `container.chart` etc.

Also recommend more semantic names (container vs div), and maybe `loader` should be `loading`.